File: src/resources/templates/basic/lexer/MyLangLexer.java
package mylang.lexer;

import org.ascore.lang.ASCLexer;

public class MyLangLexer extends ASCLexer {
    public MyLangLexer() {
        super(MyLangLexer.class.getResourceAsStream("/grammar_rules/Grammar.yaml"));
    }
}

File: src/resources/templates/basic/kotlin/lexer/MyLangLexer.kt
package mylang.lexer

import org.ascore.lang.ASCLexer

/**
 * This class is used to lex the source code. Override and edit the [lex] method to change the
 * lexing behavior.
 */
class MyLangLexer(filePath: String) : ASCLexer(MyLangLexer::class.java.getResourceAsStream(filePath))
